### PR TITLE
Yavorsky chrome to electron

### PR DESCRIPTION
**Pull request by Kilian**
Add a chromium-to-electron set of functions, continuing on from #6.

cc @yavorsky

### CODE REVIEW

**Comment by yavorsky** [comment]
@Kilian awesome, thanks!

File: README.md [hunk 1]
```diff
@@ -34,6 +34,25 @@ console.log(versions['1.4.11']);
 // returns "53.0.2785.143"
 ```
 
+#### chromiumVersions
+An object with all _major_ Chromium versions as keys and their respective major Electron version as string value.
```
Comment (yavorsky) [inline README.md]: values ?

File: build.js [hunk 2]
```diff
@@ -5,35 +5,39 @@ const fs = require('fs');
...
+    const electronVersions = {};
+    const electronFullVersions = {};
+    const chromiumVersions = {};
+    const chromiumFullVersions = {};
 
-    const makePrintable = mapping => JSON.stringify(mapping)
-                                      .replace(/,/g, ",\n\t")
-                                      .replace(/{/g, "{\n\t")
-                                      .replace(/}/g, "\n}");
+    const makePrintable = mapping => JSON.stringify(mapping, null, "\t");
```
Comment (yavorsky) [inline build.js:13]: 👍

**Review by yavorsky — APPROVED** [review]
LGTM! 👍

File: README.md (hunk 1 above)
Comment (Kilian) [reply README.md]: Reworded to remove the ambiguity (thanks @paulchaplin!)

File: README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ var e2c = require('electron-to-chromium');
 The Electron-to-Chromium object has 4 properties to use:
 
 #### versions
-An object with all _major_ Electron versions as keys and their respective major Chromium version as string value.
+An object of key-value pairs of a _major_ Electron version as the key, and the respective major Chromium version as the value.
```
Comment (existentialism) [inline README.md]: Nit:

> An object of key-value pairs **with** a _major_ Electron version as the key, and the respective _major_ Chromium version as the value.

Or even more succinctly:

> An object mapping a _major_ Electron version to its respective _major_ Chromium version.

**Pull request merged**